abaco.go

```diff
@@ -397,8 +397,10 @@ type AbacoSource struct {
 
 	groups map[GroupIndex]*AbacoGroup
 
-	readPeriod  time.Duration
-	buffersChan chan AbacoBuffersType
+	readPeriod      time.Duration
+	buffersChan     chan AbacoBuffersType
+	unwrapEnable    bool // whether to activate unwrapping
```

Comment:
This seems redundant, the `PhaseUnwrapper` struct already stores this info, it doesn't need to also be stored in the `AbacoSource`. I guess it is done so that at line line 538 in sample you can use the values inside the `Sample` function which is part of an interface, and therefore doesn't really make sense to add these values as arguments to. So I don't see a simple solution to avoid having this property. Maybe just add a comment to that effect.

Reply:
Yeah, the problem being solved here is that the `AbacoSource.Configure()` function is where we learn what the user had in mind for unwrapping enable/reset time, but the `DataSource.Sample()` function is where we need to _use_ that information. So it has to be stored in the `AbacoSource` object. I guess I can comment to this effect.